tests/legacy-cli/e2e/utils/process.ts

```diff
@@ -96,39 +96,47 @@ function _exec(options: ExecOptions, cmd: string, args: string[]): Promise<Proce
   _processes.push(childProcess);
 
   // Create the error here so the stack shows who called this function.
+  let error: Error;
+  try {
+    throw new Error();
+  } catch (e) {
+    error = e as Error;
+  }
+
+  // Return log info about the current process status
+  function envDump() {
+    return [
+      `ENV:${JSON.stringify(spawnOptions.env, null, 2)}`,
```

Comment:
This also changed but is a bit hard to see. It now outputs the env of the process being spawned, not the parent process - `spawnOptions.env` isntead of `process.env`. I think that is far more useful and probably the original intention?